chapters/traversal.md

```diff
@@ -1,83 +1,85 @@
 # Understand Jina Recursive Document Representation
 
```

Comment:
Please add a section that explains the purpose of this document. Something like. 

This guide explains how Documents are represented within the Jina framework. It is especially relevant for understanding how segmenters, crafters and rankers work.

Reply:
oh, yes. I have added your suggestion